plugins/woocommerce-admin/client/profile-wizard/steps/store-details/index.js

```diff
@@ -112,10 +110,13 @@ export class StoreDetails extends Component {
 	}
 
 	onSubmit() {
-		this.setState( {
-			showUsageModal: true,
-			skipping: false,
-		} );
+		if ( this.props.allowTracking ) {
+			this.props.skipProfiler();
```

Comment:
@moon0326 There is an edge case. When I clicked on "Continue" button and  `allowTracking = true`, I was redirected to the home page. Is this expected?

Steps to reproduce:

1. Go to OBW
2. Skip the profiler and allow tracking
3. Go to OBW again
4. Select a country
5. Click "Continue" button


https://user-images.githubusercontent.com/4344253/183330456-d015aac0-4c46-433a-8c24-e011558c2099.mov

Reply:
Thank you for catching it 👍 

The method should have been `goToNextStep` instead. Updated in https://github.com/woocommerce/woocommerce/pull/34176/commits/8089eac4c0f41fd48f8151222d41b2d3fbc986b3